app/konnect/org-management/audit-logging/webhook.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Set up an audit log webhook
+content_type: how-to
+badge: enterprise
+beta: true
+---
+
+You can use the {{site.konnect_short_name}} Organization Settings API to configure webhooks for [audit logging](/konnect/org-management/audit-logging/). 
+
+Webhooks are invoked via an HTTP request using the following retry rules:
+
+- Minimum retry wait time: 1 second
+- Maximum retry wait time: 30 seconds
+- Maximum number of retries: 4
+
+A retry is performed on connection error, server error (`500` HTTP status code), or too many requests (`429` HTTP status code).
+
+## Prerequisites
+
+Before you can push {{site.konnect_short_name}} audit logs to an external service, you need to configure the service to receive logs. 
+This configuration is specific to your vendor.
+
+You can configure a webhook into any application that supports the [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm) or raw JSON.
+
+1. Check your SIEM documentation to find out where to send CEF or raw JSON data.
+
+1. In your log collection service, configure a data collection endpoint to push logs to.
+
+1. Take note of the authorization credentials that you need to access this endpoint. {{site.konnect_short_name}} supports any HTTP authorization header type.
+
+## Create a webhook
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. From the navigation menu, open {% konnect_icon organizations %} **Organization**, then **Audit Logs Setup**.
+1. Fill in the fields in the **Setup** tab.
+   * **Region endpoint**: The external endpoint that will receive audit log messages. 
+   * **Authorization Header**: The authorization type and credential to pass to your log collection endpoint. 
+    {{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint.
+
+     For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: 
+     `"authorization":"Splunk example-token12234352535235"`.
+        
+    * **Log Format**: The output format of each log message. Can be CEF or JSON.
+1. Switch the toggle to `Enabled`, then save your webhook configuration.
+
+{% endnavtab %}
+{% navtab API %}
+
+Now that you have an external endpoint and authorization credentials, you can set up a webhook in {{site.konnect_short_name}}.
+
+Create a webhook by sending a request to the `/audit-log-webhook` endpoint with the connection details for your SIEM vendor:
+
+```sh
+curl -i -X PATCH https://global.api.konghq.com/v2/audit-log-webhook \
```

Comment:
Could we update the endpoint url in this example to be `https://us.api.konghq.com/v2/audit-log-webhook`
or `https://eu.api.konghq.com/v2/audit-log-webhook` since these are region-specific?